### ISSUE TITLE
Store waypoints and messages

### ISSUE THREAD

**Issue by sabas** [tananaev/traccar#1803]
Hi,
a nice feature to have would be the possibility to send manually to the server a text string (and additional media?) with a command - implemented on the client side with a button plus some fields to add the information to send.

I was thinking at an use case where some people surveying an area can send information and some other people can use the information in real time (emergency relief or land surveyors, in my particular situation an OpenStreetMap mapping party).

/cc @napo @piersoft

**Comment by tananaev** [comment]
Are you talking about Traccar Client? It's already possible to handle text messages on the server side.

**Comment by sabas** [comment]
Do you mean in https://github.com/tananaev/traccar/blob/master/database/changelog-3.3.xml#L132?

Yes the question was related both to the server and the client :-)

**Comment by tananaev** [comment]
Yes, all additional data will be stored in the `attributes` field.
